app/controllers/webpay_controller.rb

```diff
@@ -31,7 +31,43 @@ def commit
     render 'transaction_committed'
   end
 
-  def send_commit
+  def refund
+    @req = params.as_json
+    @token = params[:token]
+    @amount = params[:amount]
+    @resp = Transbank::Webpay::WebpayPlus::Transaction::refund(token: @token, amount: @amount)
+  end
 
+  def status
+    @req = params.as_json
+    @token = params[:token]
+    @resp = Transbank::Webpay::WebpayPlus::Transaction::status(token: @token)
   end
+
+  def mall_create
```

Comment:
Why this method is empty?

Reply:
It renders the 'mall_create' view

Reply:
you can write this as `def mall_create; end`

Reply:
I know, I just find that to be extremely ugly

Reply:
This style guide is against me, though
https://github.com/rubocop-hq/ruby-style-guide#no-single-line-methods

Reply:
😂 it's ok